models/endpoint.model.js

```diff
@@ -0,0 +1,8 @@
+const fs = require("fs/promises");
+
+exports.selectEndpoints = () => {
+  return fs.readFile(
+    `$(__dirname)/../endpoints.js`,
+    "utf-8"
+  );
+};
```

Comment:
Since you're not using selectEndpoints anymore, don't forget to delete this file!